docs/dev/development.adoc

```diff
@@ -178,15 +178,21 @@ NOTE: Refer https://github.com/golang/go/wiki/LearnTesting for Go best practices
 
 === Integration and e2e tests
 
-*Prerequisites:*
+*Prerequisites for Openshift cluster:*
 
 * A `minishift` or OpenShift environment with Service Catalog enabled:
 +
 ----
 $ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--enable=*,service-catalog,automation-service-broker,template-service-broker"
 ----
 
-* `odo` and `oc` binaries in `$PATH`.
+*Prerequisites for Kubernetes cluster:*
+
+* A `kubernetes` environment set up with single node cluster:
++
+For `kubernetes` single node cluster install link:https://kubernetes.io/docs/tasks/tools/install-minikube/[`Minikube`]
```

Comment:
`for a single node kubernetes cluster` and only `install` no need for link

Reply:
Done